corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -116,7 +128,8 @@ def test_duplicate_client_claim(self):
         response = self.client.post(self.url, {'case_id': self.case_id})
         self.assertEqual(response.status_code, 201)
         # Dup claim
-        response = self.client.post(self.url, {'case_id': self.case_id})
+        response = self.client.post(self.url, {'case_id': self.case_id},
+            HTTP_X_COMMCAREHQ_LASTSYNCTOKEN=self.synclog.synclog_id)
```

Comment:
can you add tests for duplicate claim with a synclog ID that doesn't exist and with a synclog that doesn't contain the case